pages/src/landing/marquee/marquee-logo.less

```
@@ -1,25 +1,36 @@
-#esl-logo-shield-stars > path.animate {
+#esl-logo-shield-stars > use.animate {
   animation: shine-orange 0.5s linear;
 }
 @keyframes shine-orange {
   0%, 20%, 80%, 100% {
     stroke: transparent;
   }
   50% {
-    stroke: #ffb79b;
+    stroke: #ffb79c;
   }
 }
-#esl-logo-border-inner-stars > path.animate,
-#esl-logo-border-outer-stars > path.animate,
-#esl-logo-wrench-left-stars > path.animate,
-#esl-logo-wrench-right-stars > path.animate {
+#esl-logo-border-inner-stars > use.animate,
+#esl-logo-border-outer-stars > use.animate {
   animation: shine-blue 0.5s linear;
 }
 @keyframes shine-blue {
   0%, 20%, 80%, 100% {
     stroke: transparent;
   }
   50% {
-    stroke: rgb(152, 198, 255);
+    stroke: #98C7FF;
```

It seems that lower case letters are used more often.
Maybe we should add a rule `color-hex-case: lower` to stylelint config?
```suggestion
    stroke: #98c7ff;
```